src/syscall/syscall_libc_darwin.go

```diff
@@ -43,6 +43,7 @@ const (
 	EEXIST      Errno = 0x11
 	EINTR       Errno = 0x4
 	ENOTDIR     Errno = 0x14
+	EINVAL      Errno = 22
```

Comment:
Can you make this hexadecimal to match the other constants?

Reply:
What if I covert them all to decimal? Having them in hexadecimal is a bit odd to me.

Reply:
That sounds like it would be better done in a separate PR. For this PR, perhaps you could please make it a hex value like the others?

Reply:
Sure, updated.

Reply:
I don't mind which form it is, as long as it's consistent. Feel free to make a PR to convert these to decimal if you want to :)